libraries/rush-lib/src/schemas/experiments.schema.json

```diff
@@ -53,6 +53,10 @@
     "forbidPhantomResolvableNodeModulesFolders": {
       "description": "If true, Rush will not allow node_modules in the repo folder or in parent folders.",
       "type": "boolean"
+    },
+    "enablePnpmSyncForInjectedDependenciesMeta": {
+      "description": "If true, Rush will use pnpm-sync to manage injected dependenciesMeta in PNPM.",
```

Comment:
`dependenciesMeta` is really an implementation detail.  For example, it's generally more intuitive to write about "optional peer dependencies" rather than "peer dependencies with `optional:true` specified in `dependenciesMeta`". 🙂

Suggest to shorten the name to `usePnpmSyncForInjectedDependencies`.

Reply:
Then the docs would be like:
```
(UNDER DEVELOPMENT) For certain installation problems involving peer dependencies, PNPM cannot
correctly satisfy versioning requirements without installing duplicate copies of a package inside the
node_modules folder. This poses a problem for "workspace:*" dependencies, as they are normally
installed by making a symlink to the local project source folder. PNPM's "injected dependencies" 
feature provides a model for copying the local project folder into node_modules, however copying
must occur AFTER the dependency project is built and BEFORE the consuming project starts to build.
The "pnpm-sync" tool manages this operation; see its documentation for details.

Enable this experiment if you want "rush" and "rushx" commands to resync injected dependencies
by invoking "pnpm-sync" during the build.
```